### PR TITLE
[Time-Series] Added blog-post to tips

### PR DESCRIPTION
**Pull request by elisim**
@kashif

### CODE REVIEW

**Comment by kashif** [comment]
thanks! LGTM 👍🏽

File: docs/source/en/model_doc/autoformer.md [hunk 1]
```diff
@@ -29,6 +29,9 @@ The abstract from the paper is the following:
 This model was contributed by [elisim](https://huggingface.co/elisim) and [kashif](https://huggingface.co/kashif).
 The original code can be found [here](https://github.com/thuml/Autoformer).
 
+Tips:
+- Check out the Autoformer blog-post in HuggingFace blog: [Yes, Transformers are Effective for Time Series Forecasting (+ Autoformer)](https://huggingface.co/blog/autoformer)
```
Comment (NielsRogge) [inline docs/source/en/model_doc/autoformer.md:33]: You can add this in a separate Resources section (see BERT or ViT's docs as an example)

Comment (elisim) [reply docs/source/en/model_doc/autoformer.md:36]: Done, thanks!

**Comment by kashif** [comment]
Thanks!

**Review by kashif — APPROVED** [review]

**Pull request merged**